Vue Component.sublime-syntax.yaml-macros

```diff
@@ -0,0 +1,428 @@
+      push:
+        - match: (?i)(-->)?\s*(</)(script)(>)
+          captures:
+            0: meta.tag.script.end.html
+            1: comment.block.html punctuation.definition.comment.html
+            2: punctuation.definition.tag.begin.html
+            3: entity.name.tag.script.html
+            4: punctuation.definition.tag.end.html
+          pop: true
+        - match: '(>)\s*(<!--)?'
+          captures:
+            1: meta.tag.script.begin.html punctuation.definition.tag.end.html
+            2: comment.block.html punctuation.definition.comment.html
+          embed: scope:source.livescript
+          escape: (?i)(?=(-->)?\s*</script)
+        - match: ''
+          push:
+            - meta_scope: meta.tag.script.begin.html
+            - match: '(?=>)'
+              pop: true
+            - include: tag-attributes
+    - match: (?:^\s+)?(<)((?:script))\b(?=[^>]*lang\s*=\s*(['"])ts\1?)
+      captures:
+        0: meta.tag.script.begin.html
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.script.html
+      push:
+        - match: (?i)(-->)?\s*(</)(script)(>)
+          captures:
+            0: meta.tag.script.end.html
+            1: comment.block.html punctuation.definition.comment.html
+            2: punctuation.definition.tag.begin.html
+            3: entity.name.tag.script.html
+            4: punctuation.definition.tag.end.html
+          pop: true
+        - match: '(>)\s*(<!--)?'
+          captures:
+            1: meta.tag.script.begin.html punctuation.definition.tag.end.html
+            2: comment.block.html punctuation.definition.comment.html
+          embed: scope:source.ts
+          escape: (?i)(?=(-->)?\s*</script)
+        - match: ''
+          push:
+            - meta_scope: meta.tag.script.begin.html
+            - match: '(?=>)'
+              pop: true
+            - include: tag-attributes
+    - match: (?:^\s+)?(<)((?:script))\b(?=[^>]*)
+      captures:
+        0: meta.tag.script.begin.html
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.script.html
+      push:
+        - match: (?i)(-->)?\s*(</)(script)(>)
+          captures:
+            0: meta.tag.script.end.html
+            1: comment.block.html punctuation.definition.comment.html
+            2: punctuation.definition.tag.begin.html
+            3: entity.name.tag.script.html
+            4: punctuation.definition.tag.end.html
+          pop: true
+        - match: '(>)\s*(<!--)?'
+          captures:
+            1: meta.tag.script.begin.html punctuation.definition.tag.end.html
+            2: comment.block.html punctuation.definition.comment.html
+          embed: scope:source.js
+          escape: (?i)(?=(-->)?\s*</script)
+        - match: ''
+          push:
+            - meta_scope: meta.tag.script.begin.html
+            - match: '(?=>)'
+              pop: true
+            - include: tag-attributes
+    - match: (?:^\s+)?(<)((?:style))\b(?=[^>]*lang\s*=\s*(['"])sass\1?)
```

Comment:
Same comment about the backreference and the possibly-unnecessary whitespace matching.

Reply:
Removed the whitespace matching. thank you.